pygfx/objects/_lights.py

```diff
@@ -468,11 +482,6 @@ def decay(self, value):
 
 
 # shadows
-
-_look_target = Vector3()
-_proj_screen_matrix = Matrix4()
```

Comment:
I see all these global linalg objects have vanished, and no global ndarrays have come in their place, which is great! can you confirm that there are no global linalg object instances remaining in the pygfx codebase?

Reply:
I deleted the entire `pygfx.linalg` folder, so if there were any globals of that kind they should complain upon import :D

Apart from that, I haven't consciously re-introduced any module- or class-level variables that aren't constant, so there shouldn't be none left, yes.